wolox-books/src/views/SignUp.vue

```diff
@@ -11,18 +11,25 @@
     input.input(type='text' v-model='lastName')
     label.label
       | Email
-    input.input(type='text' v-model='email')
+    input.input(type='text' v-model='$v.email.$model')
+    .error(v-if="$v.email.$error")
+      | Email is required
     label.label
       | Password
-    input.input(type='password' v-model='password')
-    a.input.signup-button(href='javascript:void(0)' v-on:click='submitForm')
-      | Sign Up
+    input.input(type='password' v-model='$v.password.$model')
+    .error(v-if="$v.password.$error")
+      | Password is required
+    input.input.signup-button(v-on:click='submitForm' type='submit' value='Sign Up')
     .login-container
       a.input.login-button(href='javascript:void(0)')
         | Login
 </template>
 
+<script src="vuelidate/dist/vuelidate.min.js"></script>
```

Comment:
This line is not needed, as you are using Vuelidate as a plugin (`Vue.use(Vuelidate)`).